README.md

```diff
@@ -1,2 +1,139 @@
 # github-security-jira
-Github Action for integrating Security Alerts with JIRA
+GitHub Action for mapping security alerts to Jira tickets.
+
+
+## Setup
+
+This action is designed to run in conjunction with another action. In summary, you need the following pieces set up to sync alerts with Jira:
+
+1. Two repo secrets containing a GitHub access token and a Jira API token, respectively.
```

Comment:
Actions get a token per default, isn't that sufficient? https://help.github.com/en/actions/automating-your-workflow-with-github-actions/authenticating-with-the-github_token#about-the-github_token-secret

Reply:
AFAICT the default token does not have access to security alerts.